.github/workflows/release-test.yml

```diff
@@ -3,154 +3,80 @@ name: Release test Workflow
-        with:
-          cmd: npx nx run-many --target=paths --all --parallel=2
-          affectedCmd: npx nx affected --target=paths
-          isDev: ${{ env.IS_DEV }}
-  test-unit:
+  release_test:
     runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run && github.event.workflow_run.conclusion == 'success' || true }}
+    environment: 'Release test'
+    env:
+      GH_TOKEN: ${{ secrets.GH_TOKEN }}
+      GIT_USER: ${{ vars.GIT_USER }}
+      GIT_EMAIL: ${{ vars.GIT_EMAIL }}
+      RELEASE_BRANCH: ${{ github.event.workflow_run.head_branch || github.ref }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          ref: ${{ env.GITHUB_REF }}
-          fetch-depth: 0
-          submodules: recursive
-      - uses: ./.github/actions/init
-      - name: Unit Tests
-        uses: ./.github/actions/run-optimal
-        with:
-          cmd: |
-            npx nx run-many --target=test --all --parallel=2
-            node tools/coverage-reports/create-merged-layer-coverage-report.js libs/base
-            node tools/coverage-reports/create-merged-layer-coverage-report.js libs/domain
-            node tools/coverage-reports/create-merged-layer-coverage-report.js libs/platform
-          affectedCmd: npx nx affected:test
-          isDev: ${{ env.IS_DEV }}
-  test-visual-pr:
-    if: ${{ github.actor != 'dependabot[bot]' && github.event_name == 'pull_request' }}
-    runs-on: ubuntu-latest
-    steps:
+          node-version: 18
       - uses: actions/checkout@v3
         with:
-          ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0
+          token: ${{ env.GH_TOKEN }}
+          ref: ${{ env.RELEASE_BRANCH }}
           submodules: recursive
-      - uses: ./.github/actions/init
-      - name: Visual Regression Tests
-        env:
-          CHROMATIC_PROJECT_TOKEN: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
-          CHROMATIC_SHA: ${{ github.event.pull_request.head.sha }}
-          CHROMATIC_BRANCH: ${{ github.event.pull_request.head.ref }}
-        run: npx nx affected --target=visual-regression-pr
-  test-visual-dev:
-    if: ${{ github.event_name == 'push' }}
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
         with:
-          ref: ${{ env.GITHUB_REF }}
-          fetch-depth: 0
-          submodules: recursive
-      - uses: ./.github/actions/init
-      - name: Visual Regression Tests
+          path: ~/.npm
+          key: npm-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            npm-node-
+      - name: Auth Git
+        run: |
+          git config user.email "${{ env.GIT_EMAIL }}"
+          git config user.name "${{ env.GIT_USER }}"
+      - name: Install Deps
+        run: npm ci
+      - name: Build packages
+        run: npx nx run-many --target=build --all --parallel=2 --exclude testfront, storybook, fulfillment, storefront
```

Comment:
The list of excluded packages is different here - https://github.com/spryker/oryx/pull/251/files#diff-fab5d050d343dd5b95a7a1e24ae6bea6c4dfbd5a714e432d0588caa6fbbfae46R20 (should we adjust these after merges?)

Reply:
Maybe create a small follow up ticket to not forget about it)

Reply:
Yep, I'll adjust.